frontend/debug.log

```diff
@@ -0,0 +1 @@
+[1105/143717.120:ERROR:directory_reader_win.cc(43)] FindFirstFile: �� ������� ��� ����� �� ���� �� ��������� ��� ����������� �������� ������. (0x3)
```

Comment:
It might be better not to keep log files on our GitHub repo.
You can delete this file and add perhaps a line with "debug.log" in .gitignore to avoid committing this file again in the future.